source/includes/steps-create-agg-pipeline.yaml

```
@@ -50,5 +50,27 @@ content: |
 
          .. figure:: /images/compass/agg-builder-exclude-stage-example.png
             :alt: Aggregation Builder exclude stage example
-...
+---
+title: Run Pipeline.
```

The other steps in this procedure use sentence case, so I think we should use that here as well:

```suggestion
title: Run the pipeline.
```

---

Done.